tests/end-to-end/backup/pkg/common/common.go

```diff
@@ -81,22 +81,26 @@ func RunTest(t *testing.T, mode TestMode) {
 	myApiGatewayScenarioTest, err := ory.NewApiGatewayTest()
 	fatalOnError(t, err, "while creating structure for Api-Gateway test")
 
+	myEventMeshTest, err := eventmesh.NewEventMeshTest()
+	fatalOnError(t, err, "while creating structure for event mesh test")
+
 	//rafterTest := rafter.NewRafterTest(client)
 
 	backupTests := []e2eTest{
 		{enabled: true, backupTest: myPrometheusTest},
-		{enabled: true, backupTest: myGrafanaTest},
+		{enabled: false, backupTest: myGrafanaTest}, //disabled due to flakiness
```

Comment:
is there an issue to get these tests rectified?

Reply:
https://github.com/kyma-project/kyma/issues/7298